src/amo/components/SearchContextCard/styles.scss

```diff
@@ -1,7 +1,12 @@
 @import '~amo/css/styles';
 
 .SearchContextCard-header {
-  font-size: $font-size-l;
+  font-size: $font-size-m;
+  font-weight: normal;
```

Comment:
Why do you need `font-weight` here?

Reply:
Umm because for the image attached for expectation https://github.com/mozilla/addons-frontend/issues/7530#issue-405651445 the weight looks normal.

If its not needed let me know i will remove it.

Reply:
oops my bad making some changes sorry.